frontend/src/methods/features.ts

```diff
@@ -31,6 +33,33 @@ export const setupWorkloadProxyingEnabledFeatureWatch = (): Ref<boolean> => {
   })
 }
 
+let userPilotInitialized = false
+
+export const initializeUserPilot = async () => {
+  if (!currentUser.value) {
+    return
+  }
```

Comment:
wouldn't it be nicer to pass the user as a param to the function and keep it a bit more pure? we have it in the place we call this initialise anyway

or is there a reason we do it this way?